.gitignore

```diff
@@ -1,5 +1,4 @@
 *.o
-*.a
```

Comment:
If there is a .a file lying around in my repo, then I would want to know about it when I run git status.

I don't see why one would want to gitignore .a files.

Reply:
So you are okay with me removing it from .gitignore?

Reply:
Yes

Reply:
Oh, sorry about that, didn't see the red color.